### DISCUSSION TITLE
How to Download opentelemetry-collector-contrib.exe

### DISCUSSION THREAD

**Discussion by codebysubha** [open-telemetry/opentelemetry-collector-contrib#34932] [Q&A] [answered]
Hello Team,

I want to test the **Filelog Receiver component** to see how it works, but I'm having trouble finding where to download the otelcontrib.exe. Can someone guide me on where to find and download the exe file?

Thank you!

**Comment by arminru** [comment, marked as answer]
@codebysubha You can find the binaries for each collector release in the "Assets" section here:
https://github.com/open-telemetry/opentelemetry-collector-releases/releases

E.g. for https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.108.0 you might be looking for this compressed archive:
https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.108.0/otelcol-contrib_0.108.0_windows_386.tar.gz

**Reply by codebysubha** [reply]
thank you !!
